dll/win32/msgina/shutdown.c

```diff
@@ -15,6 +15,7 @@
 #define WLX_SHUTDOWN_STATE_LOGOFF       0x01
 #define WLX_SHUTDOWN_STATE_POWER_OFF    0x02
 #define WLX_SHUTDOWN_STATE_REBOOT       0x04
+#define WLX_SHUTDOWN_STATE_NT_MODE      0x08
```

Comment:
It kind of sounds like these values are matching Windows functionality and 0x08/0x20/0x80 exist but we don't implement them. In that case it might be better to add the new flag at the end (0x100), if the data type used to store these is large enough.

Reply:
Moved to 0x100